Train.json

```
@@ -4807,6 +4821,57 @@
 				{"name": "passengers", "value": 88}
 			]
 		},
+		{
+			"id": 8071217,
+			"group": 1,
+			"name": "zweiachsiger Reko-Wagen",
+			"shortcut": "Baag",
+			"speed": 90,
+			"weight": 17,
+			"force": 0,
+			"length": 13,
+			"drive": 0,
+			"reliability": 0.9,
+			"cost": 60000,
+			"exchangeTime": 80,
+			"capacity": [
+				{"name": "passengers", "value": 48 }
+			]
+		},
+		{
+			"id": 80152019,
+			"group": 1,
+			"name": "dreiachsiger Reko-Wagen",
+			"shortcut": "Baag",
```

```suggestion
			"shortcut": "Bag",
```

Da sonst der unterschied zwischen 2-achsigem und 3 achsigem nicht soo groß ist, _könnte_ man einen davon auch zum Ba(a)gtr machen (für Traglasten - eine Bankreihe bzw. 2 "Abteile" fehlen, also nur noch 32 Sitzplätze, genug Platz für Kinderwagen/Fahrräder etc. falls das mal als Frachtart eingeführt werden sollte).

---

Ja, das würde hier den Unterschied deutlicher machen.